crawler/helpers/general_helpers.py

```diff
@@ -296,22 +296,22 @@ def get_dart_well_index(coordinate: Optional[str]) -> Optional[int]:
     return None
 
 
-def is_sample_positive(sample):
+def is_sample_positive(sample: SampleDoc) -> bool:
     return sample.get(FIELD_RESULT, False) == RESULT_VALUE_POSITIVE
 
 
-def is_sample_important_or_positive(sample):
+def is_sample_important_or_positive(sample: SampleDoc) -> bool:
     return is_sample_positive(sample) or is_sample_important(sample)
 
 
-def is_sample_important(sample):
+def is_sample_important(sample: SampleDoc) -> bool:
     return (sample.get(FIELD_MUST_SEQUENCE, False) is True) or (
         sample.get(FIELD_PREFERENTIALLY_SEQUENCE, False) is True
     )
```

Comment:
This function is not going to be needed anymore and could be removed.

Reply:
Also, the README.md would need an update to remove the definition of 'important' and describe the changes on what is pickable now